VarLife.md

```diff
@@ -0,0 +1,59 @@
+## OTCA Metapixel
+
+<img src="http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png"/>
+(Source: [http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png](http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png))
```

Comment:
`[Source](http://www.conwaylife.com/w/images/3/3a/Otcametapixel.png)` would be better, or just leave it off because the next link references the source page on the wiki.